airbyte-api/src/main/openapi/config.yaml

```diff
@@ -2692,6 +2692,24 @@ components:
       properties:
         sourceId:
           $ref: "#/components/schemas/SourceId"
+    SourceCloneRequestBody:
+      type: object
+      required:
+        - sourceCloneId
+      properties:
+        sourceCloneId:
+          $ref: "#/components/schemas/SourceId"
+        sourceConfiguration:
+          $ref: "#/components/schemas/SourceCloneConfiguration"
+    SourceCloneConfiguration:
+      type: object
+      properties:
+        connectionConfiguration:
+          $ref: "#/components/schemas/SourceConfiguration"
+        workspaceId:
```

Comment:
why do we want to let them update workspace id? won't that potentially give other workspaces access to other connectors?

Reply:
That's true. I thought what if we want to clone a source into other workspace?

But yeah we can remove that feature for now 😅